docs/source/daml/reference/packages.rst

```diff
@@ -0,0 +1,53 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Reference: DAML packages
+########################
+
+This page gives reference information on DAML package dependencies:
+
+.. contents:: :local:
+
+DAML archives
+*************
+
+When a DAML project is build with ``daml build``, build artifacts are generated in the hidden
+directory ``.daml/dist/`` relative to the project root directory. The main build artifact of a
+project is the `DAML archive`, recoginzed by the ``.dar`` file ending. DAML archives can be deployed
+on a ledger (see :ref:`deploy <daml-ref-deploy>`) or can be imported into other projects as a package dependency.
+
+Importing DAML archives
+***********************
+
+A DAML project can import DAML package dependencies. Note that currently there is no package
+managing tooling for DAML packages and the process is manual. To import a package ``Bar`` in project
+``Foo``, add the file path of the ``Bar`` DAML archive to the ``dependencies`` section of the
+`daml.yaml` project file:
+
+.. code-block:: yaml
+
+  sdk-version: 0.0.0
+  name: foo
+  source: daml/foo.daml
+  version: 1.0.0
+  exposed-modules:
+    - Some.Module
+    - Some.Other.Module
+  dependencies:
+    - daml-prim
+    - daml-stdlib
+    - /home/johndoe/bar/.daml/dist/bar.dar
+
+The import path needs to be the absolute path pointing to the created DAML archive of the ``bar``
```

Comment:
IIRC @lucianojoublanc-da tried relative paths and that worked.

Reply:
Ah, even better.